tools/abel/abel.py

```diff
@@ -82,7 +82,7 @@ def _spreadsheet_workflow(spreadsheet_id: str,
     handler.RunValidation(
         filenames=[BC_EXPORT_PATH],
         report_filename=INSTANCE_VALIDATOR_LOG_PATH,
-        modified_types_filepath=ontology_path,
+        default_types_filepath=ontology_path,
```

Comment:
this is very dirty hack to bypass the warnings.
What if the user didnt provide an ontology path here? aren't you deprecating the feature to allow users to run abel with a modified local ontology?

Reply:
Well the user never had the ability to pass in a local ontology. Previously I was passing in the DB ontology as a modified ontology which felt like a dirtier hack. 

Re your comment below: I'm not forcing the user to set the ontology path. I gave the default ontology path a default value of the path from the instance validtor. If a developer wants to use the instance validator's `RunValidation` method, now they have the option to pass in the DB ontology from a different location as well as a modified ontology. I'm almost positive that `constants.ONTOLOGY_ROOT` cannot be modified for sure outside of the instance validator.